go.mod

```diff
@@ -229,7 +229,7 @@ replace (
 	github.com/crypto-org-chain/cronos/store => ./store
 	github.com/crypto-org-chain/cronos/versiondb => ./versiondb
 	github.com/ethereum/go-ethereum => github.com/crypto-org-chain/go-ethereum v1.10.19-deepcopy-jumptable
-	github.com/evmos/ethermint => github.com/crypto-org-chain/ethermint v0.20.10-cronos.0.20230515032938-bad31ab4475d
+	github.com/evmos/ethermint => github.com/crypto-org-chain/ethermint v0.20.10-cronos.0.20230526070717-0ea7b86532a1
```

Comment:
it's tagged as `v0.20.11-cronos`

Reply:
https://github.com/crypto-org-chain/ethermint/compare/bad31ab4475d...crypto-org-chain:ethermint:v0.20.11-cronos